Anastasia_PCS/Homework #1.js

```diff
@@ -0,0 +1,252 @@
+//Homework #1
+
+//(Due Monday 7/20)
+
+//You need to turn in this homework by cloning this repo, making a new branch with your solutions, 
+//pushing to github, and issuing a pull request. For details, follow the instructions here.//
+
+//Please also read the learning objectives for this week.
+
+//1) (30% of total time) Predict the output of each of the following expressions, and explain 
+//the rules which determine that answer. (Hint: The rules for operator == are complicated, 
+//but do your best!)
+
+//If the output depends on the value or type of variable x, identify the conditions 
+//(what x is) when the expression will output true (or false, if that's simpler). Assume the cases 
+//are independent, and x is reset to an unknown value before each.
+
+//Some of these are tricky! Don't trust your first instinct.//
+
+a) "1" == 1	//TRUE - compares value only
+
+b) "1" === 1 //FALSE - compares value & type
+
+c) x == 'x' //TRUE - compares value only
+
```

Comment:
Thanks so much for looking this over!

On Wed, Jul 22, 2015 at 1:13 PM, Eric Gray <notifications@github.com> wrote:

> In Anastasia_PCS/Homework #1.js
> <https://github.com/portlandcodeschool/jse-fall15-1/pull/4#discussion_r35257964>
> :
>
> > +//1) (30% of total time) Predict the output of each of the following expressions, and explain
> > +//the rules which determine that answer. (Hint: The rules for operator == are complicated,
> > +//but do your best!)
> > +
> > +//If the output depends on the value or type of variable x, identify the conditions
> > +//(what x is) when the expression will output true (or false, if that's simpler). Assume the cases
> > +//are independent, and x is reset to an unknown value before each.
> > +
> > +//Some of these are tricky! Don't trust your first instinct.//
> > +
> > +a) "1" == 1	//TRUE - compares value only
> > +
> > +b) "1" === 1 //FALSE - compares value & type
> > +
> > +c) x == 'x' //TRUE - compares value only
> > +
>
> In this case what we're actually looking at is a comparison of variable x
> to string 'x'. The value of variable x could be any number of things,
> whereas the value of string 'x' is always going to be 'x', resulting in a
> false return.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/portlandcodeschool/jse-fall15-1/pull/4/files#r35257964>
> .
>